chain-abci/tests/tx_validation.rs

```diff
@@ -189,9 +188,12 @@ fn prepate_init_tx(
     let old_tx_id = old_tx.id();
 
     let mut inittx = db.transaction();
-    mock_client
-        .local_tx_store
-        .insert(old_tx_id, TxWithOutputs::Transfer(old_tx));
+    // FIXME
```

Comment:
`FIXME` what?

Reply:
https://github.com/crypto-com/chain/issues/885 -- I'll add the mention